odin/adapters/dummy.py

```diff
@@ -135,3 +141,81 @@ def cleanup(self):
         """
         logging.debug("DummyAdapter cleanup: resetting background test counter")
         self.background_task_counter = 0
+
+
+class IacDummyAdapter(ApiAdapter):
+    """Dummy adapter class for the Inter Adapter Communication changes.
+
+    This dummy adapter impelements the basic operations of GET and PUT,
+    and allows another adapter to interact with it via these methods.
+    """
+
+    def __init__(self, **kwargs):
+        """Initialize the dummy target adapter.
+
+        Create the adapter using the base adapter class.
+        Create an empty dictionary to store the references to other loaded adapters.
+        """
+
+        super(IacDummyAdapter, self).__init__(**kwargs)
+        self.adapters = {}
+
+        logging.debug("IAC Dummy Adapter Loaded")
+
+    @response_types("application/json", default="application/json")
+    def get(self, path, request):
+        """Handle a HTTP GET Request
+
+        Call the get method of each other adapter that is loaded and return the responses
+        in a dictionary.
+        """
+        logging.debug("IAC DUMMY GET")
```

Comment:
Shouty-shouty debug messages :wink:

Reply:
Debug's gotta get your attention somehow ;)

Reply:
True, but that's through the sheer force of your coding brilliance, not gratuitous use of all-caps!

Reply:
:laughing: